test/confirm-blocklist.test.js

```diff
@@ -0,0 +1,37 @@
+const test = require("tape");
+const { testDomain } = require("./test.util.js");
+const sqlite3 = require("sqlite3").verbose();
+const db = new sqlite3.Database("./tranco.db");
+
+function runTests(config) {
+  test("check config blocklist against top Tranco domains", async (t) => {
+    let foundWhitelistedDomains = "";
```

Comment:
Let's make this an array so we don't need to do string concat on line 16

`let foundWhitelistedDomains: string[] = [];`